mmaction/apis/inference.py

```diff
@@ -57,21 +47,14 @@ def init_recognizer(config,
     return model
 
 
-def inference_recognizer(model,
-                         video_path,
-                         label_path,
-                         use_frames=False,
-                         outputs=None,
-                         as_tensor=True):
-    """Inference a video with the detector.
+def inference_recognizer(model, video, outputs=None, as_tensor=True):
+    """Inference a video with the recognizer.
```

Comment:
since the API is changed, better to use `kwargs` to get `label_path` and `use_frames` and provide deprecated infos